File: week-02/day-01/draw_diagonal.py
# Write a program that reads a number from the standard input, then draws a
# square like this:
#
#
# %%%%%
# %%  %
# % % %
# %  %%
# %   %
# %%%%%
#
# The square should have as many lines as the number was
N = int(input("Give me a number! "))

def square():
    for i in range(N):
        if i == 0 or i == (N-1):
            print("%" * N)
        else:
            print("%" + (i-1)* " " + "%" + (N-2-i) * " " + "%")

square()

File: week-07/day-01/draw_diagonal.js
'use strict';

var lineCount = 6;


// Write a program that draws a
// square like this:
//
//
// %%%%%
// %%  %
// % % %
// %  %%
// %   %
// %%%%%
//
// The square should have as many lines as lineCount is

for (let i = 0; i <= lineCount-1; i++) {
  if (i === 0 || i === lineCount-1) {
    console.log(Array(lineCount+1).join("%"))
  } else {
    console.log("%" + Array(i).join(" ") + "%" + Array(lineCount-1-i).join(" ") + "%");
  }
}